core/simulations/configs/vaultLiquidation.ts

```diff
@@ -0,0 +1,112 @@
+import { warpTime, resetNetworkAndSetupWallet, addDaiToBalance, addMkrToBalance } from '../../helpers/hardhat';
+import { Simulation } from '../types';
+import { CollateralConfig } from '../../src/types';
+import prompts from 'prompts';
+import COLLATERALS from '../../src/constants/COLLATERALS';
+import BigNumber from '../../src/bignumber';
+import { collectStabilityFees, fetchVault, liquidateVault } from '../../src/vaults';
+import { HARDHAT_PUBLIC_KEY, TEST_NETWORK } from '../../helpers/constants';
+import createVaultForCollateral, { getCollateralAmountInVat } from '../steps/createVaultForCollateral';
+
+const getSupportedCollaterals = () => {
+    const supportedCollaterals: Record<string, CollateralConfig> = JSON.parse(JSON.stringify(COLLATERALS));
+    delete supportedCollaterals['CRVV1ETHSTETH-A'];
+    return supportedCollaterals;
+};
+
+export const SUPPORTED_COLLATERALS = getSupportedCollaterals();
```

Comment:
It's a bad practice to assign something to a variable and then export it. I suggest that you export `getLiquidatableCollateralTypes` function that returns list of types and is used in other files
```suggestion
const UNSUPPORTED_COLLATERAL_TYPES = ['CRVV1ETHSTETH-A'];

export const getLiquidatableCollateralTypes = () => {
    return getAllCollateralTypes().filter(collateralType => !UNSUPPORTED_COLLATERAL_TYPES.includes(collateralType));
};
```